backend/hitas/models/indices.py

```diff
@@ -22,27 +22,27 @@ class Meta:
 # 'Markkinahintaindeksi'
 class MarketPriceIndex(AbstractIndex):
     class Meta:
-        verbose_name = _("Market price index for apartments constructed in January 2005 or after")
-        verbose_name_plural = _("Market price indices for apartment constructed in January 2005 or after")
+        verbose_name = _("Market price index for apartments constructed before January 2011")
+        verbose_name_plural = _("Market price indices for apartment constructed before January 2011")
 
 
-class MarketPriceIndexPre2005(AbstractIndex):
+class MarketPriceIndex2005Equal100(AbstractIndex):
     class Meta:
-        verbose_name = _("Market price index for apartments constructed before January 2005")
-        verbose_name_plural = _("Market price indices for apartment constructed before January 2005")
+        verbose_name = _("Market price index for apartments constructed in January 2011 onwards")
+        verbose_name_plural = _("Market price indices for apartment constructed in January 2011 onwards")
```

Comment:
Seems like a typo here 🤔 
`Market price index for apartments constructed in January 2011 __and__* onwards`?

Reply:
I don't think `and` is necessary here.  [Few examples](https://en.bab.la/sentences/english/onwards)